.github/workflows/run_tests.yml

```diff
@@ -177,11 +178,23 @@ jobs:
           - os: "ubuntu-22.04"
             python-version: "3.12"
             arch: "arm"
+            docker_engine: "docker"
             unit-tesseract: "base"
+          # Run tests using Podman
+          - os: "ubuntu-24.04"
+            python-version: "3.12"
+            arch: "x64"
+            docker_engine: "podman"
+            unit-tesseract: "base"
+          - os: "ubuntu-24.04"
+            python-version: "3.12"
+            arch: "x64"
+            docker_engine: "podman"
+            unit-tesseract: "vectoradd_jax"
 
       fail-fast: false
 
-    runs-on: ${{ matrix.arch == 'x64' && matrix.os == 'ubuntu-22.04' && 'ubuntu-22.04' || matrix.arch == 'arm' && matrix.os == 'ubuntu-22.04' && 'linux-arm64-ubuntu2204' }}
+    runs-on: ${{ matrix.arch == 'x64' && matrix.os == 'ubuntu-22.04' && 'ubuntu-22.04' || matrix.arch == 'arm' && matrix.os == 'ubuntu-22.04' && 'linux-arm64-ubuntu2204' ||  matrix.arch == 'x64' && matrix.os == 'ubuntu-24.04' && 'ubuntu-24.04'}}
```

Comment:
Thoughts about upgrading to 24.04 in general (for all tests)?

Reply:
I was gonna ask you the same :smile:

Should we test with both versions to ensure compatibility with older systems as well?

Reply:
I'm fine if we go to 24.04 unconditionally